connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerConnectorTest.java

```diff
@@ -134,7 +134,30 @@ public void testInitializeFailure(ConnectorType connectorType) {
 
         verifyInitialize();
         verify(listener).onFailure(CONNECTOR, exception);
-        verifyCleanShutdown(false);
+        verifyCleanShutdown(true);
+    }
+
+    @ParameterizedTest
+    @EnumSource(value = ConnectorType.class, names = {"SOURCE", "SINK"})
+    public void testInitializeAndStopFailure() {
```

Comment:
ParameterizedTests take a parameter, look at the other tests in this class.
This test recently got migrated to junit 5 so it's been refactored slightly.

Reply:
I mean the tests are still not written correctly, because they don't take a `ConnectorType connectorType` parameter, or call `startup(connectorType)`

Reply:
That indicates that at least one test failed. Look in the top right corner for "Tests".